components/TextFields/tests/unit/TextFieldControllerLegacyTests.swift

```diff
@@ -246,7 +246,7 @@ class TextFieldControllerDefaultLegacyTests: XCTestCase {
 
     controller.helperText = "Helper"
     textField.sizeToFit()
-    XCTAssertEqual(textField.frame.height, 84.5)
+    XCTAssertTrue(MDCCGFloatEqual(MDCFloor(textField.frame.height), 84.0))
```

Comment:
This test is failing on iPad 2 simulator when I run the entire Unit Test suite, but passes when I run it individually. I'm a little concerned that there is some interaction between the tests.

Reply:
Let's try this version.